resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/DeTwineStrings.kt

```diff
@@ -0,0 +1,149 @@
+        searchResultsCount = { count ->
+            when (count) {
+                1 -> "$count Ergebnis"
+                else -> "$count Ergebnisse"
+            }
+        },
+        bookmark = "Lesezeichen",
+        bookmarks = "Lesezeichen",
+        bookmarksPlaceholder = "Mit Lesezeichen versehene Beiträge werden hier angezeigt 🔖",
+        settings = "Einstellungen",
+        moreMenuOptions = "Weitere Menüoptionen",
+        settingsHeaderBehaviour = "Verhalten",
+        settingsHeaderFeedback = "Feedback und Fehlerberichte",
+        settingsHeaderOpml = "OPML",
+        settingsBrowserTypeTitle = "Verwenden Sie den In-App-Browser",
+        settingsBrowserTypeSubtitle =
+        "Wenn diese Option deaktiviert ist, werden Links in Ihrem Standardbrowser geöffnet.",
+        settingsEnableBlurTitle = "Aktivieren Sie die Unschärfe auf der Startseite",
+        settingsEnableBlurSubtitle =
+        "Wenn deaktiviert, werden Farbverläufe statt unscharfer Bilder angezeigt. Kann die Leistung verbessern.",
+        settingsShowUnreadCountTitle = "Anzahl der ungelesenen Beiträge anzeigen",
+        settingsShowUnreadCountSubtitle =
+        "Wenn diese Option deaktiviert ist, wird die Anzahl der ungelesenen Beiträge nicht im Feed-Blatt angezeigt",
+        settingsReportIssue = "Ein Problem melden",
+        settingsVersion = { versionName, versionCode -> "$versionName ($versionCode)" },
+        settingsAboutTitle = "Um Twine",
+        settingsAboutSubtitle = "Lernen Sie die Autoren kennen",
+        settingsOpmlImport = "Importieren",
+        settingsOpmlExport = "Export",
+        settingsOpmlImporting = { progress -> "Importieren.. $progress%" },
+        settingsOpmlExporting = { progress -> "Exportieren.. $progress%" },
+        settingsOpmlCancel = "Stornieren",
+        settingsPostsDeletionPeriodTitle = "Gelesene Beiträge löschen",
+        settingsPostsDeletionPeriodOneWeek = "1 Woche",
+        settingsPostsDeletionPeriodOneMonth = "1 Monat",
+        settingsPostsDeletionPeriodThreeMonths = "3 Monate",
+        settingsPostsDeletionPeriodSixMonths = "6 Monate",
+        settingsPostsDeletionPeriodOneYear = "1 Jahr",
+        settingsShowReaderViewTitle = "Enable reader view",
+        settingsShowReaderViewSubtitle = "Articles will be shown in a simplified, easy-to-read view",
+        feeds = "Einspeisungen",
+        editFeeds = "Feeds bearbeiten",
+        comments = "Kommentare",
+        about = "Um",
+        aboutRoleDeveloper = "Entwickler",
+        aboutRoleDesigner = "Designerr",
+        aboutSocialTwitter = "Twitter",
+        aboutSocialThreads = "Threads",
+        aboutSocialGitHub = "GitHub",
+        aboutSocialWebsite = "Webseite",
+        feedsSearchHint = "Such-Feeds",
+        allFeeds = "Alle Feeds",
+        pinnedFeeds = "Angepinnt",
+        openWebsite = "Website öffnen",
+        markAllAsRead = "Alles als gelesen markieren",
+        noNewPosts = "Keine neuen Inhalte",
```

Comment:
Posts would be "Beiträge" in other strings, so I would go for "Keine neuen Beiträge." (No new posts.)